spec/models/comment_spec.rb

```diff
@@ -1,5 +1,47 @@
 require 'rails_helper'
 
 RSpec.describe Comment, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'Validations For the Comment model' do
```

Comment:
- [ ] Kindly use some positive test cases, for the `comment` model test.

Reply:
Hey @Deepakdanger please show me an example of how to implement positive test cases?

Reply:
I have Implemented it even though you are not willing to help and is not part of the requirements.